.github/workflows/visual-diff.yml

```diff
@@ -3,7 +3,7 @@ on: pull_request
 jobs:
   build:
     name: Visual Diff
-    timeout-minutes: 20
+    timeout-minutes: 30
```

Comment:
😭

Reply:
I know. The focus reset in input color tests caused them all the shift ever so slightly and get re-generated, which caused the tests to timeout... I don't think I'll ever _not_ be working on this.

Reply:
I'm considering leaving this at 30 minutes, since it seems like any more than 5 diffs and the runs actually timeout instead of creating the PR with the changes.